features/options/routed/help.feature

```diff
@@ -49,7 +50,8 @@ Feature: osrm-routed command line options: help
         And stdout should contain "--port"
         And stdout should contain "--threads"
         And stdout should contain "--sharedmemory"
-        And stdout should contain 22 lines
+        And stdout should contain "--max_locations_distance_table"
```

Comment:
just remember to update wiki and let people know in due time. could break peoples toolchains otherwise :-)